src/intrinsics/prepack/global.js

```diff
@@ -398,6 +398,25 @@ export default function(realm: Realm): void {
     })
   );
 
+  // Helper function for Prepack developers inspect a value
```

Comment:
Helper function letting Prepack developers inspect a value using a debugger.

Reply:
I commented on this earlier, not sure where my comment went. I mentioned that it's probably best to remove both lines of comments as they don't add any real value.

Reply:
@trueadm Oops - Not sure why I didn't see your comments. Perhaps you approved, and added the nits afterwards.

Reply:
Yeah, there's some strange things happening to comments in the latest Github update. Sebastian wrote loads of comments to one of my PRs from a few weeks back and I never got them either, he had to remake them all.